IdentityCore/src/requests/broker/MSIDSSOExtensionRequestDelegate.m

```diff
@@ -52,7 +53,15 @@ - (void)authorizationController:(__unused ASAuthorizationController *)controller
     assert(self.completionBlock);
     if (!self.completionBlock) return;
     
-    self.completionBlock(nil, error);
+    NSError *underlyingError = error.userInfo[NSUnderlyingErrorKey];
```

Comment:
should we log the error we receive here?

Reply:
done